src/test/ceph-erasure-code-tool/test_ceph-erasure-code-tool.sh

```diff
@@ -5,22 +5,22 @@ mkdir $TMPDIR
 trap "rm -fr $TMPDIR" 0
 
 ceph-erasure-code-tool test-plugin-exists INVALID_PLUGIN && exit 1
-ceph-erasure-code-tool test-plugin-exists jerasure
+ceph-erasure-code-tool test-plugin-exists isa
 
 ceph-erasure-code-tool validate-profile \
-                       plugin=jerasure,technique=reed_sol_van,k=2,m=1
+                       plugin=isa,technique=reed_sol_van,k=2,m=1
```

Comment:
They are not 1:1 transferrable. I've added a tool that prints out the hash of each chunk here https://github.com/ceph/ceph/pull/58377

Here are chunks for ISA and Jerasure using reed_sol_van with k=3 m=3

```
[root@9d9c7969fd9a build]# cat plugin_comparison/output
489715dfc67930395b840e6f62f87310  plugin_comparison/plugin=isa stripe-width=256 k=3 m=3 technique=reed_sol_van/0
f2dfcedc0cbd3b8247267941e44ede2b  plugin_comparison/plugin=isa stripe-width=256 k=3 m=3 technique=reed_sol_van/1
3b86c3580ddae1fd8a2bf7b0d94ef931  plugin_comparison/plugin=isa stripe-width=256 k=3 m=3 technique=reed_sol_van/2
3041babbcf88ebd486a4e103e7cdc4ae  plugin_comparison/plugin=isa stripe-width=256 k=3 m=3 technique=reed_sol_van/3
72af377096a80419bc2d607031ad3404  plugin_comparison/plugin=isa stripe-width=256 k=3 m=3 technique=reed_sol_van/4
4d88897a89854d5732fd094b401d3107  plugin_comparison/plugin=isa stripe-width=256 k=3 m=3 technique=reed_sol_van/5

489715dfc67930395b840e6f62f87310  plugin_comparison/plugin=jerasure stripe-width=256 k=3 m=3 technique=reed_sol_van/0
f2dfcedc0cbd3b8247267941e44ede2b  plugin_comparison/plugin=jerasure stripe-width=256 k=3 m=3 technique=reed_sol_van/1
3b86c3580ddae1fd8a2bf7b0d94ef931  plugin_comparison/plugin=jerasure stripe-width=256 k=3 m=3 technique=reed_sol_van/2
3041babbcf88ebd486a4e103e7cdc4ae  plugin_comparison/plugin=jerasure stripe-width=256 k=3 m=3 technique=reed_sol_van/3
858b48eb0c113ee3f217e91a426fc4fd  plugin_comparison/plugin=jerasure stripe-width=256 k=3 m=3 technique=reed_sol_van/4
b50e4bdf05f80ba7cab1164505cd6e87  plugin_comparison/plugin=jerasure stripe-width=256 k=3 m=3 technique=reed_sol_van/5
```
The first coding chunk is the same because we are using an xor optimisation for that one, but the second and third coding chunks are different.

So if we change the default plugin, it should be for new pools only.